staff_features/accessions/accession_delete.feature

```diff
@@ -16,7 +16,7 @@ Feature: Accession Delete
      When the user clicks on 'Delete'
       And the user clicks on 'Delete' in the confirm popup
      Then the Accessions page is displayed
-      And the 'Accession deleted' message is displayed
+      And the 'Accession' deleted message is displayed
```

Comment:
Very minor point, but I wonder if this was intentional? If yes then line 12 needs a similar edit. It seems to me however that `'Accession deleted'` is fine.

Reply:
Good point Brian. Yes, this was intentional, however, I agree it causes confusion.

Initially, we tested the accessions without filling in the title field, that's why I created a concrete match for `the 'Accession deleted' message is displayed`. If the title is filled in, then the message is: `the 'Accession <Accession Title Here> deleted' message is displayed`, which is covered by regular expression matching [here](https://github.com/archivesspace/e2e-tests/blob/b751944a3009044dec514d0c527220c75349195b/staff_features/shared/step_definitions.rb#L62-L78).

I have removed the concrete matching and updated all the messages to be matched by regular expressions.